source/Minable.h

```diff
@@ -51,6 +51,18 @@ class Minable : public Body {
 		double toughness = 1.;
 	};
 
+	class LiveEffect {
```

Comment:
I think this can be private? It doesn't look like anything outside the Minable class ever acccesses it.